tests/Export/Adapters/ExportItemAdapterTest.php

```diff
@@ -62,9 +70,10 @@ public function testExceptionIsNotThrownForVariantWithNoCategories(): void
         $product = $this->createTestProduct([
             'id' => $id,
             'categories' => [],
+            'variantListingConfig' => ['displayParent' => null],
```

Comment:
Probably makes sense to add this as default value, it should not make issues with the existing tests